build/versionize-docs.sh

```diff
@@ -60,6 +62,9 @@ for doc in "${mdfiles[@]}"; do
   # Replace /HEAD in html preview links with /NEW_VERSION.
   $SED -ri -e "s|(${HTML_PREVIEW_PREFIX}/HEAD)|${HTML_PREVIEW_PREFIX}/${NEW_VERSION}|" "${doc}"
 
+  # Replace <REPLACE-WITH-RELEASE-VERSION> with NEW_VERSION
+  $SED -ri -e "s|${DIRCTORY_KEY_WORDS}|${NEW_VERSION}|" "${doc}"
```

Comment:
Add "|g" at the end (for replace all occurrences).
I am doing that at other places in this file in my PR.